storm-client/src/jvm/org/apache/storm/Config.java

```diff
@@ -205,6 +205,12 @@
     public static final String TOPOLOGY_TASKS = "topology.tasks";
 
     /**
+     * A map of resources used by each component e.g {"cpu" : 200.0. "onheap.memory.mb": 256.0, "gpu" : 0.5 }
```

Comment:
Is this the default resources?  Is this intended for users to actually modify?  A bit more explanation here would be good.